client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/index.tsx

```diff
@@ -404,11 +404,7 @@ const designSetup: Step = function DesignSetup( { navigation, flow } ) {
 
 	// When the intent is build, we can potentially show the generated design picker.
 	// Don't render until we've fetched the generated designs from the backend.
-	if (
-		enabledGeneratedDesigns &&
-		( isLoadingGeneratedDesigns || ! siteVerticalId ) &&
-		! isAnchorSite
```

Comment:
I think we can disable `useStarterDesignsGeneratedQuery` when `isAnchorSite = true` in a follow up PR.